### ISSUE TITLE
How could I get the image's name and tag after using lthe loadImageCmd() function?

### ISSUE THREAD

**Issue by fighterhit** [docker-java/docker-java#1110]
How could I get the image's name and tag after using loadImageCmd() function? I have seen the test case but there is only a simple[ use case](https://github.com/docker-java/docker-java/blob/master/src/test/java/com/github/dockerjava/cmd/LoadImageCmdIT.java) which the loadImageCmd(uploadStream) function didn't return anything  after exec.But I need to know  the image's name and tag so that I could push the resulting image to my private registry.Is there any solution? Thanks!

**Comment by fighterhit** [comment]
> Have you resolved this problem, I got the same question, could you help me

I have seen docker-java source code and found that it really return nothing. So I have to uncompress the image tar archive and get image's name and tag in manifest.json.

**Comment by fighterhit** [comment]
> It's a good ideal. Have you push the images to a private or public registry? Colud you give me an example?thanks

You could use `org.apache.commons.compress` lib to uncompress the tar and then read the  manifest.json.

**Comment by huyunhao** [comment]
Thank you, I've resolved it

**Comment by lbzwoaini** [comment]
you can use creatImageCmd, which can load image from tar file and named it at both time. The response of this command contains the image id.

**Comment by Tianny** [comment]
> creatImageCmd

Hi，when I use createImageCmd, I recently encountered some difficulties. The image created by createImageCmd from tarball looks like different from the original image in tarball. For example, I defined CMD instructions in the original image in tarball, but when I run the created image, the container produces error.

I think something wrong when user createImageCmd. The new image is not the same as the image in tarball.

Hope your reply. THX.
